packages/seal-solr-adapter/src/SolrIndexer.php

```diff
@@ -29,6 +29,7 @@ public function __construct(
         private readonly Client $client,
     ) {
         $this->marshaller = new FlattenMarshaller(
+            dateFormat: 'Y-m-d\TH:i:s\Z',
```

Comment:
Solr Expect a specific date format, While it did work with our `c` format it did remove the timezone incorrectly and so the filters didn't work. We are now converting it into correct values.